libs/core-styles/src/lib/_imports/settings/font.css

```diff
@@ -18,14 +18,15 @@ Reference:
 
 Styleguide Settings.CustomProperties.Font
 */
+@import url('../generics/fonts.css');
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap');
 
 :root {
   /* Family */
-  /* NOTE: Clients are expected to overwrite these values via their CSS */
-  --global-font-family--sans: sans-serif;
-  --global-font-family--serif: serif;
-  --global-font-family--mono: monospace;
-
+  --global-font-family--sans--cms: "Benton Sans", "Roboto", sans-serif;
+  --global-font-family--sans--portal: "Roboto", sans-serif;
+  --global-font-family--serif: Times, sans-serif;
+  --global-font-family--mono: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
```

Comment:
ℹ️ Portal uses Roboto. CMS uses Benton Sans and Roboto. Now TUP uses Benton Sans to (for the login page). So, share them all.